src/actions/ui.js

```diff
@@ -11,6 +11,13 @@ export const editorFocusedRequestedLine = createAction(
   'EDITOR_FOCUSED_REQUESTED_LINE',
 );
 
+export const editorsUpdatedFlex = createAction(
+  'EDITORS_UPDATED_FLEX',
+  flex => ({flex}),
+);
```

Comment:
Generally my naming convention for actions is:

* If an action is directly triggered by a user interaction, use an imperative (e.g. `UPDATE_EDITOR_FLEX`)
* If an action is triggered by the outcome of an asynchronous operation, use the past tense passive voice (e.g. `GIST_IMPORTED`)